ShellScript/Shell-Unix-Generic.sublime-syntax

```diff
@@ -418,6 +418,8 @@ contexts:
       push:
         - meta_scope: meta.scope.if-block.shell
         - match: \b(fi)\b
+          captures:
+            1: keyword.control.shell
```

Comment:
Is there a reason why you're using captures here? Am I missing something? 
Cause I would simply use scope on it, like `scope: keyword.control.shell`.